test/boulder-test-config.json

```diff
@@ -32,12 +32,10 @@
 
   "wfe": {
     "baseURL": "http://localhost:4300",
-    "listenAddress": "127.0.0.1:4300"
+    "listenAddress": "0.0.0.0:4300"
```

Comment:
Specifically for the test-config, fwiw. I don't care if the `start.sh` and `start_amqp.sh` scripts do it by default (that's why the override is there), `test.sh` is all localhost anyway.